packages/template-retail-react-app/app/utils/locale.js

```diff
@@ -17,16 +17,31 @@ import fetch from 'cross-fetch'
  * @returns {Promise<Object>} The messages (compiled in AST format) in the given locale.
  *      If the translation file is not found, return an empty object (so react-intl would fall back to the inline messages)
  */
-export const fetchTranslations = async (locale) => {
+export const fetchTranslations = async (locale, origin) => {
     const targetLocale =
         typeof window === 'undefined'
             ? process.env.USE_PSEUDOLOCALE === 'true'
                 ? 'en-XA'
                 : locale
             : locale
 
+    const isServerSide = typeof window === 'undefined'
+
     try {
-        const file = `${getAppOrigin()}${getAssetUrl(
+        if (isServerSide && process.env.NODE_ENV === 'production') {
```

Comment:
FYI currently, `fetchTranslations` is called only on the _server_ side. So this is the if-condition that will be reached.

Reply:
We called the fetchTranslations from within the App component. See https://github.com/SalesforceCommerceCloud/pwa-kit/blob/88361bb0593492dd854de4272c0711b75535a3f3/packages/template-retail-react-app/app/components/_app/index.jsx#L169